src/types/search.d.ts

```diff
@@ -36,7 +37,7 @@ declare class Search {
 
   getSearchResults(
     query: string,
-    parameters?: SearchParameters,
+    parameters?: Omit<SearchParameters, 'preFilterExpression' | 'qsParam'>,
```

Comment:
Why are we omitting these here 🤔 ?

Reply:
I'll trade you my [mistake from yesterday](https://github.com/Constructor-io/constructorio-client-javascript/pull/250) and we'll forget both happened